Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -292,7 +310,8 @@ def process_incident_create_message(demisto_user: dict, message: str, request_bo
         server_links: dict = demisto.demistoUrls()
         server_link: str = server_links.get('server', '')
         data = f"Successfully created incident {created_incident.get('name', '')}.\n" \
-               f"View it on: {server_link}/#/WarRoom/{created_incident.get('id', '')}"
+               f"View it on: {server_link + '/#' if not is_demisto_version_ge('8.0.0') else server_link}/" \
```

Comment:
I would do the server_link assignment in a line before 312
server_link = server_link + '/#' if not is_demisto.... else server_link.
and in line 313 leave it as it was